src/Element/NoteFooter.tsx

```diff
@@ -140,15 +179,16 @@ export default function NoteFooter(props: NoteFooterProps) {
             <FontAwesomeIcon icon={faReply} />
           </div>
         </div>
+        <Menu menuButton={<div className="reaction-pill">
```

Comment:
<img width="769" alt="Screenshot 2023-01-21 at 09 06 59" src="https://user-images.githubusercontent.com/92299987/213857725-b692132c-f146-459b-8a9f-0144b693c8ed.png">

wdyt about removing a bit of x padding? looks a bit wide.

Reply:
![image](https://user-images.githubusercontent.com/1172179/213865840-ec01a227-04e0-4e40-ace3-dbcff2b4e94a.png)